paparazzi-gradle-plugin/src/test/projects/supports-dynamic-feature-modules/app/build.gradle

```diff
@@ -15,3 +17,9 @@ android {
 
   dynamicFeatures = [':dynamic_feature']
 }
+
+dependencies {
+  implementation files('libs/external.aar')
+}
+
```

Comment:
Unlike `library` module, `application` module does not generate `symbol_list_with_package_name/[variant]/package-aware-r.txt` when there is no dependency. So I have to add one dependency in the test. 

In reality, `application` module will always have dependencies, so there is no issue about that. We do not have to check if `symbol_list_with_package_name/[variant]/package-aware-r.txt` exists